xknx/telegram.py

```diff
@@ -1,12 +1,127 @@
 import time
 from .colors import Colors
-from .address import Address
+from .address import Address,AddressType
+class CouldNotParseCEMI(Exception):
+    def __init__(self):
+        pass
+
+    def __str__(self):
+        return "CouldNotParseCEMI"
+		
+class CEMIMessage():
+    """Representation of a CEMI message."""
+
+    CMD_GROUP_READ = 1
+    CMD_GROUP_WRITE = 2
+    CMD_GROUP_RESPONSE = 3
+    CMD_UNKNOWN = 0xff
+
+    code = 0
```

Comment:
sollten diese variablen nicht in `__init__` . Wenn ich das richig verstehe, sind diese Variablen global definiert... was spätestens beim Multithreading kaputt geht.